### PR TITLE
Prepare v0.44.0 release

### PR DESCRIPTION
**Pull request by philipaconrad**
Note: This release commit rolls in a small fix for the build/release tooling: an upgrade for the changelog-generator script from Python 2 -> Python 3. The script can now handle unicode in commit messages without breaking.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,163 @@
...
+final result set is ever modified, reducing memory allocations and GC pressure.
+The `union` builtin is now about 15-30% faster than the equivalent operation in
+pure Rego.
+
+### Bulk Prefix and Suffix String Collection Matching Builtins
+
+This release introduces two new builtins, optimized for bulk matching of string
+prefixes and suffixes: `strings.any_prefix_match`, and
+`strings.any_suffix_match`.
+
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:49]: Nit: Seems to be repeated below.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,163 @@
...
+- docs: Update port and bundle folder for GraphQL tutorial
+- docs: Document that function overloading is unsupported
+- docs: Fixing related_resources annotations example ([#4982](https://github.com/open-policy-agent/opa/issues/4982)) reported by @humbertoc-silva
+- docs: Fixing typo in metadata, issue #5018 (#5019) ([#5018](https://github.com/open-policy-agent/opa/issues/5018)) authored by @cimin0 reported by @cimin0
+
+### Website + Ecosystem
+
+- Update links to opa-kafka-plugin (#5021)
+- Add OCI documentation (#4781) (authored by @carabasdaniel)
+- Fix broken link (#5071) (authored by @unautre)
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:133]: Can you add more context here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,163 @@
...
+  - build(deps): bump github.com/containerd/containerd from 1.6.6 -> 1.6.8
+  - build(deps): bump github.com/go-ini/ini from 1.66.6 -> 1.67.0
+  - build(deps): bump github.com/prometheus/client_golang
+  - build(deps): bump google.golang.org/grpc from 1.48.0 -> 1.49.0
+  - build(deps): bump tj-actions/changed-files from 28.0.0 -> 29.0.3
+
+- Dependency removals:
+  - internal: Vendor gqlparser library (#5067) ([#5065](https://github.com/open-policy-agent/opa/issues/5065)) authored by @philipaconrad reported by @vikstrous2
+
+### Miscellaneous
```
Comment (ashutosh-narkar) [inline CHANGELOG.md:160]: Multiple `Misc` sections

**Review by ashutosh-narkar — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (philipaconrad) [reply CHANGELOG.md]: Thanks for the catch! I saw that one a few moments ago, and have force-pushed a patch removing the duplicated block.

File: CHANGELOG.md (hunk 3 above)
Comment (philipaconrad) [reply CHANGELOG.md:154]: Ah, nuts. A leftover from the changelog generator script. I'll patch that out shortly. Thanks for catching that! :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (philipaconrad) [reply CHANGELOG.md:127]: That's a particularly egregious commit message; I'll flesh it out shortly.

File: CHANGELOG.md (hunk 3 above)
Comment (philipaconrad) [reply CHANGELOG.md]: Rolled this into the latest version.

File: CHANGELOG.md (hunk 2 above)
Comment (philipaconrad) [reply CHANGELOG.md]: It is done. :+1:

**Comment by philipaconrad** [comment]
@ashutosh-narkar Will merge when green.

**Pull request merged**